### PR TITLE
#167313376 User should be able to bookmark an article

### PR DESCRIPTION
**Pull request by minega25**
#### What does this PR do?
 User should be able to bookmark an article
#### Description of Task to be completed?
Users should be able to bookmark articles for reading later via a bookmark tab, button or link
#### How should this be manually tested?
Clone this branch and navigate to view single article page. Click the bookmark article button.
#### What are the relevant pivotal tracker stories?
[#167313376](https://www.pivotaltracker.com/story/show/167313376)
#### Screenshots (if appropriate)
Screen Shot 2019-11-05 at 08 32 57
Screen Shot 2019-11-05 at 08 33 02
Screen Shot 2019-11-05 at 08 33 09
Screen Shot 2019-11-05 at 08 33 16

### CODE REVIEW

**Review by habinezadalvan — APPROVED** [review]
LGTM

File: src/components/article/viewArticleComponents.js [hunk 1]
```diff
@@ -171,18 +173,17 @@ ArticleContent.defaultProps = {
   description: '',
   body: '',
 };
-
 RecentArticles.propTypes = {
-  onChange: PropTypes.func,
-  current: PropTypes.number,
-  total: PropTypes.number,
-  articles: PropTypes.array,
+  onChange: PropTypes.string,
```
Comment (SokoPaulSokool) [inline src/components/article/viewArticleComponents.js]: on change is a function not a string

File: src/components/article/viewArticleComponents.js [hunk 2]
```diff
@@ -171,18 +173,17 @@ ArticleContent.defaultProps = {
...
+  current: PropTypes.string,
+  total: PropTypes.string,
+  articles: PropTypes.string,
```
Comment (SokoPaulSokool) [inline src/components/article/viewArticleComponents.js]: You should leave these the way they were

File: src/components/article/viewArticleComponents.js [hunk 3]
```diff
@@ -171,18 +173,17 @@ ArticleContent.defaultProps = {
...
 };
 RecentArticles.defaultProps = {
-  onChange: () => ({}),
-  current: 0,
-  total: 0,
-  articles: [],
+  onChange: '',
+  current: '',
+  total: '',
+  articles: '',
```
Comment (SokoPaulSokool) [inline src/components/article/viewArticleComponents.js]: You should leave these the way they were

**Review by SokoPaulSokool — CHANGES_REQUESTED** [review]

File: src/components/article/viewArticleComponents.js (hunk 2 above)
Comment (minega25) [reply src/components/article/viewArticleComponents.js]: I think these were changed in the last PR. I dont know why they show as if I am the one that changed them

File: src/components/article/viewArticleComponents.js (hunk 3 above)
Comment (minega25) [reply src/components/article/viewArticleComponents.js]: Honestly speaking 😕, there is no motive behind me changing these. Let me check in the view single article the way they were

Comment (minega25) [reply src/components/article/viewArticleComponents.js]: What do you think went wrong here @SokoPaulSokool.

**Review by SokoPaulSokool — APPROVED** [review]
Good work

**Pull request merged**